assets/todo.json

```diff
@@ -2,10 +2,10 @@
     "name": "todo",
     "description": "A simple Todo application with a back-end implementation in every language, and an interactive tutorial for the policy language. Provides a head start for learning how to integrate Aserto into your own app.",
     "assets": {
-        "manifest": "citadel/manifest.yaml",
+        "manifest": "todo/manifest.yaml",
         "policy": {
             "name": "todo",
-            "resource": "ghcr.io/aserto-policies/policy-todo:latest"
+            "resource": "ghcr.io/aserto-policies/policy-todo:3.0.0"
```

Comment:
should we use policy-todo:latest?

Reply:
I prefer to be explicit about what version is used in production. This lets us rollout new versions and test them without worrying about breaking prod.